src/main/java/data/Account.java

```diff
@@ -32,6 +33,17 @@
   public static final String MATCHING_MODE_KEY = "matching_mode";
   public static final String ACCOUNT_ID_KEY = "account_id";
   public static final String AGGREGATION_MODE_KEY = "aggregation_mode";
+  
+  // names for HttpServletRequest conversion
+  public static final String VENDOR_ID = "vendorID";
```

Comment:
I'd really like to avoid having duplicate copies of these constants everywhere. Can we just keep them in one place?

Reply:
(maybe as a follow-up cleanup CL to handle this across the entire codebase)